types/src/chapter.rs

```diff
@@ -0,0 +1,21 @@
+use uuid::Uuid;
+use {
+    crate::*,
+};
+
+#[derive(Debug, Clone)]
+pub struct Chapter {
+    pub id: Option<Uuid>,
+    
+    pub url_slug: String,
+
+    pub index_in_parent: i64,
+
+    pub chapter_name: String,
+
+    pub document_title: Option<String>,
+
+    pub document_id: Option<DocumentId>,
+
+    pub wordpress_id: Option<i64>,
+}
```

Comment:
Make sure to run the code formatter. Outside of your editor, you can run `cargo fmt` to do this across the project.